### PR TITLE
Week 1 Quiz Submission from mphan12-stat660

### PR DESCRIPTION
**Pull request by mphan12-stat697**
Resubmission with advise changes

### CODE REVIEW

File: week-01/README.md [hunk 1]
```diff
@@ -17,49 +17,73 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Structure Quiz, Problem 1]
+Question (mphan12-stat660): When posting screenshots, should we screenshot the entire desktop or only relevant snap-its of our screen?
+Answer:
```
Comment (prof-lankham) [inline week-01/README.md]: Please do not include if not providing an answer

File: week-01/README.md [hunk 2]
```diff
@@ -17,49 +17,73 @@ The instructor will then review the pull request and make comments should furthe
 
 
 [Course Structure Quiz, Problem 1]
+Question (mphan12-stat660): When posting screenshots, should we screenshot the entire desktop or only relevant snap-its of our screen?
```
Comment (prof-lankham) [inline week-01/README.md]: All questions/answers need to begin with an asterisk and a space, in order to render as a bulleted list

File: week-01/README.md [hunk 3]
```diff
@@ -17,49 +17,73 @@ The instructor will then review the pull request and make comments should furthe
...
 
 [Course Structure Quiz, Problem 8]
+Question (mphan12-stat660): What is the maximum extra credit allowance?
+Answer: 
 
 
 
 [Course Structure Quiz, Problem 9]
+Question (mphan12-stat660): Can we still email you at isaiah.lankham@csueastbay.edu?
+Answer (mphan12-stat660): Yes
```
Comment (prof-lankham) [inline week-01/README.md]: Please give more than one-word answers

**Review by prof-lankham — CHANGES_REQUESTED** [review]
Formatting of contributions incorrect, and answers should be informative. Please review all instructions, update the PR, and DM the instructor in Slack with the PR URL when ready for re-evaluation.

**Review by mphan12-stat697 — COMMENTED** [review]
I've made the following changes:
* Bullet for list format
* Remove lines with no answer
* Adjust answers to more than yes and no.

**Pull request merged**